src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/exception/PaymentServiceException.java

```diff
@@ -0,0 +1,22 @@
+package ca.bc.gov.open.pssg.rsbc.dps.paymentservice.exception;
+
+/**
+ * 
+ * @author smillar
```

Comment:
same question as above about your github id

Reply:
same response. I use the same author all through the code so I will change it.